slack-lint-checks/src/main/java/slack/lint/denylistedapis/DenyListedApiDetector.kt

```diff
@@ -45,176 +45,18 @@ import slack.lint.denylistedapis.DenyListedEntry.Companion.MatchAll
  */
 internal class DenyListedApiDetector : Detector(), SourceCodeScanner, XmlScanner {
 
-  override fun getApplicableUastTypes() = CONFIG.applicableTypes()
```

Comment:
Can you clean up some of these extraneous refactorings? Would like to keep this addition limited to just the new Calendar entry unless there's some functional change required to make that work

Reply:
Certainly! I've removed the commits that introduced the extraneous refactorings locally and will force-push soon.